src/explore-education-statistics-admin/src/components/DraggableItem.module.scss

```diff
@@ -0,0 +1,59 @@
+@import '~govuk-frontend/govuk/base';
+
+.draggable {
+  background: govuk-colour('white');
+  border-bottom: 1px solid govuk-colour('dark-grey');
+  margin-bottom: 0;
+  padding: govuk-spacing(2) govuk-spacing(1) govuk-spacing(2) 40px;
```

Comment:
Don't have to, but it might be good to bump up the vertical padding slightly to give this a little more space. Feels quite narrow currently!